### ISSUE TITLE
No toleration present in hcloud-csi-node daemonset

### ISSUE THREAD

**Issue by jekakm** [hetznercloud/csi-driver#119]
I noticed that hcloud-csi-node daemonset not have any tolerations set. So if I use taints in cluster for some node, csi agent will not scheduled on it. I think this daemonset must pass any taints, like k8s system components (kube-proxy for example) or any another csi drivers do this in this way:

```
tolerations:
- effect: NoExecute
  operator: Exists
- effect: NoSchedule
  operator: Exists
- key: CriticalAddonsOnly
  operator: Exists
```

So it allow to pass any taint keys and volumes will mounts correctly on all nodes.

**Comment by jekakm** [comment]
for example aws csi do this in this way:
https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/deploy/kubernetes/base/node.yaml#L36

**Comment by Troyhy** [comment]
We have similar issue.

**Comment by toabi** [comment]
I noticed the same issue when I wanted to create a dedicated node for a workload. 

If I give the node a taint I cannot shedule workloads which use hetzer volumes on it because… "x node(s) had volume node affinity conflict".

**Comment by LKaemmerling** [comment]
I'm happy to announce that we just released v1.4.0 of our CSI driver which includes this. The new container should be available in a couple of minutes.
